include/rpp.h

```diff
@@ -102,38 +102,12 @@ typedef void (*rppDeallocatorFunction)(void* context, void* memory);
 extern "C" {
 #endif
 
-/*! \brief Returns error string.
- * \details Returns a NULL terminated character string of the passed error code.
- * \param [in] error Error status of rppStatus_t type.
- * \ingroup group_rpp
- * \return <tt>\ref const char*</tt>
- * \returns Returns an error string among those listed under \ref rppStatus_t.
- */
-extern "C" SHARED_PUBLIC const char* rppGetErrorString(rppStatus_t error);
-
-/*! \brief Creates RPP handle for HOST single image processing - To be deprecated.
- * \details Function to create a RPP handle. To be called in the beginning to initialize the RPP environment.
- * \param [in] handle A pointer to RPP handle of type <tt> \ref rppHandle_t</tt>.
- * \ingroup group_rpp
- * \return A <tt> \ref rppStatus_t</tt> enumeration.
- * \retval rppStatusSuccess
- * \retval rppStatusNotInitialized
- * \retval rppStatusInvalidValue
- * \retval rppStatusBadParm
- * \retval rppStatusAllocFailed
- * \retval rppStatusInternalError
- * \retval rppStatusNotImplemented
- * \retval rppStatusUnknownError
- * \retval rppStatusUnsupportedOp
- * \deprecated
- */
-extern "C" SHARED_PUBLIC rppStatus_t rppCreate(rppHandle_t* handle);
-
-/*! \brief Creates RPP handle for HOST batch processing.
- * \details Function to create a RPP handle for a batch. To be called in the beginning to initialize the RPP environment.
+/*! \brief Creates RPP handle for HOST/HIP/OCL backend batch processing.
+ * \details Function to create a RPP handle, and the necessary host/device memory allocations.
  * \param [in] handle A pointer to RPP handle of type <tt> \ref rppHandle_t</tt>.
  * \param [in] nBatchSize Batch size.
- * \param [in] numThreads Number of threads to be used for OpenMP pragma.
+ * \param [in] numThreadsOrStream A pointer to Number of threads or stream An accelerator queue of type <tt> \ref rppAcceleratorQueue_t</tt> (hipStream_t for HIP and cl_command_queue for OpenCL).
```

Comment:
I suggest using seperate parameters for numCPUThreads and stream. COmbining them into single parameter will cause ambiguity for the user. If the use is not using that, it can pass nullptr. For ex. host calls will set null for stream. If number of threads is set 0, it should use a default for CPU and ignore for GPU

Reply:
Done